src/GenFacades/GenFacades.Core/GenFacades.cs

```diff
@@ -666,8 +666,14 @@ public override void RewriteChildren(Assembly assembly)
                 // Add reference to seed core assembly up-front so that we keep the same order as the C# compiler.
                 assembly.AssemblyReferences.Add(_seedCoreAssemblyReference);
 
-                // Remove all type definitions except for the "<Module>" type.
+                // Remove all type definitions except for the "<Module>" type. Remove all fields and methods from it.
                 assembly.AllTypes.RemoveAll(t => t.Name.Value != "<Module>");
+                NamespaceTypeDefinition moduleType = assembly.AllTypes.SingleOrDefault(t => t.Name.Value == "<Module>") as NamespaceTypeDefinition;
```

Comment:
Depending on the number of types it is probably more efficient to grab the "<Module>" type first then call Clear and then add it back to the AllTypes.